### PR TITLE
Feat/component content

### LINKED ISSUE #70: Content component
# Docs

- [x] Do after [docs boilerplate](https://github.com/epicmaxco/vuestic-ui/issues/17) is done.

# Notes
- [x] Just wrap existing `content` class into component. We don't need to document global class usage, only component itself.
- [x] Move demos and docs to component scope.

# Refs
* [old issue](https://github.com/epicmaxco/vuestic-admin/issues/374)

### PR DESCRIPTION
**Pull request by Kreezag**
## Description

close #70 
add va-content component

## Markup:

<details>

```vue
<va-content>
 <h1>H1</h1>
 <h3>H2</h2>
 ...
 <pre>Text</pre>
 <blockquote>Text</blockquote>
</va-content>
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: docs/components/VaContent.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# va-content
+Component update children styles
```
Comment (jean-moldovan) [inline docs/components/VaContent.md]: We may want to add that this component applies vuestic specific typography and table styles.

File: src/components/vuestic-sass/global/typography-link-style.demo.vue [hunk 2]
```diff
@@ -1,37 +0,0 @@
-<template>
```
Comment (asvae) [inline src/components/vuestic-sass/global/typography-link-style.demo.vue]: And why was this file deleted?

Comment (Kreezag) [reply src/components/vuestic-sass/global/typography-link-style.demo.vue]: I think this file is unnecessary. 
This file does not provide useful information

File: docs/components/VaContent.md (hunk 1 above)
Comment (Kreezag) [reply docs/components/VaContent.md]: Sorry, you can explain in more detail?

File: src/components/vuestic-sass/global/typography-link-style.demo.vue (hunk 2 above)
Comment (asvae) [reply src/components/vuestic-sass/global/typography-link-style.demo.vue]: @Kreezag is this still not clear?

File: src/components/vuestic-components/va-content/VaContent.vue [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+</template>
+
+<script>
+export default {
+  name: 'VaContent',
+}
+</script>
+
+<style lang="scss">
+  @import '../../vuestic-sass/resources/resources';
```
Comment (asvae) [inline src/components/vuestic-components/va-content/VaContent.vue]: Tab.

File: src/components/vuestic-components/va-content/VaContent.vue [hunk 4]
```diff
@@ -0,0 +1,114 @@
+<template>
+  <div class="content">
```
Comment (asvae) [inline src/components/vuestic-components/va-content/VaContent.vue]: We want `.content` class and `va-content` component.
Those should be handled separately. You can use mixin or extent to abstract away class, but `.content` definitely doesn't belong to va-content.

Also naming. VaContent component should have class `va-content`.

File: src/components/vuestic-components/va-content/VaContent.demo.vue [hunk 5]
```diff
@@ -1,14 +1,14 @@
 <template>
   <VbDemo>
     <VbCard title="with content class (h1-h6)">
```
Comment (asvae) [inline src/components/vuestic-components/va-content/VaContent.demo.vue]: Title is incorrect.

File: docs/components/VaContent.md (hunk 1 above)
Comment (asvae) [reply docs/components/VaContent.md]: Documentation on this one will be handled later.
